### DISCUSSION TITLE
Actually can i install Ckan on an Windows Operating System?

### DISCUSSION THREAD

**Discussion by Aggeloslou** [ckan/ckan#7703] [Q&A] [answered]
Actually, can I install Ckan on a Windows Operating System?

**Comment by pdelboca** [comment, marked as answer]
Hello! 

CKAN is being developed and tested in gnu/linux environments and it does not provide support for Windows.

Said that, there is people who had installed and work on it in Windows. Maybe in our [Gitter Channel](https://app.gitter.im/#/room/#ckan_chat:gitter.im) you can find someone.

Bests
